### PR TITLE
Do not set UIWindow bounds

### LINKED ISSUE #14: setup(mainWindow:) shouldn't set UIWindow's bounds on iOS9+
When setting UIWindow's bounds to `UIScreen.main.bounds` it breaks apps on iPads with iOS 9 and higher. This is caused because of the multitasking feature introduced in iOS 9. Setting those bound make the app appear under the status bar. Having just `makeKeyAndVisible` results in correct behavior.

### PR DESCRIPTION
**Pull request by benjohnde**
Closes #14 

Just grabbed the low hanging fruit and adjusted the deployment target as well as the removal of settings the bounds to `UIScreen.bounds` programmatically.

### CODE REVIEW

File: Sources/SuperDelegate.swift [hunk 1]
```diff
@@ -55,7 +55,6 @@ open class SuperDelegate: NSObject, UIApplicationDelegate {
             return
         }
         
-        mainWindow.frame = UIScreen.main.bounds
```
Comment (dfed) [inline Sources/SuperDelegate.swift]: I just tested this line on an iOS 8 simulator (plus iPad and a few others sims). I'm shocked this works. But, it does! So let's back out the platform bump.

**Review by dfed — CHANGES_REQUESTED** [review]
The side-effect of this change is that calling `setup(mainWindow:)` no longer sets the window size. So if clients have code that assumes the VC installed in the window has a non-zero size, that code will break.

That makes this change a breaking one. Before merging this, I'd like to do some investigation into when the window is set, and see if we can build API around that lifecycle.

That said, it looks like View Controllers installed in the size-zero window are the right size. So, maybe we're fine? Let's do a bit more investigation before we merge this.

**Comment by benjohnde** [comment]
Removed the platform bump. Ready to merge from my side.

This PR fixes #14.

**Comment by dfed** [comment]
This looks good to me! I'd like some confirmation from known adopters that this doesn't break anything on their side (it shouldn't, but I'm a little nervous changing this). @mthole @efirestone can you take a look and see if this change would impact your apps?

**Pull request closed without merging**